src/modules/shared.mjs

```diff
@@ -20,7 +21,7 @@ export function logEvent(name, parameters) {
 }
 
 export function setupAnalytics() {
-  if (window.location.hostname === "secure.dol.state.nj.us") {
+  if (isProduction()) {
```

Comment:
Dust: wondering if its good to also set `analyticsScript.async = true` here as it's async in [gtag docs](https://developers.google.com/tag-platform/gtagjs/install)

Reply:
This part was actually intentional. `async=true` means that we fetch the analytics in parallel with parsing the HTML page. This makes sense when you are loading the page with the `script` tag already in the HTML. In this case, since we're being hacky and adding this `script` tag _after_ loading the page, I thought it makes more sense to keep this synchronous, since the parsing step is probably already done. It feels like there's less assumptions about the page load when you keep it synchronous here.

Reply:
Gotcha! I wasn't quite sure about this one so its good to hear the explanation :)